packages/merge-queue/README.md

```diff
@@ -0,0 +1,28 @@
+# merge-queue
+
+This is a small bot for managing merge queue for repositories.
+
+## How to use
+
+Add `merge-queue:add` label to PRs you want to automatically merge. The bot will
+put the pull requests in a queue and try to merge them in order. This bot is
+especially useful if you configure pull requests need to be up to date before
+merge.
+
+
+## Deploying and testing
```

Comment:
Please discuss how an attacker might attempt to use this bot to merge malicious code into the code base.

Reply:
The attacker needs to have permission to add a label to a PR. In this regard, the situation is the same as merge-on-green.

Reply:
Added some notes